definitions/v7/mojblink.yml

```diff
@@ -0,0 +1,224 @@
+    - { id: 8, cat: PC/Games, desc: "GAMES PC" }
+    - { id: 7, cat: PC/Games, desc: "GAMES PCRip" }
+    - { id: 13, cat: Console/PS3, desc: "GAMES Playstation" }
+    - { id: 124, cat: Console/PS4, desc: "GAMES PS4" }
+    - { id: 16, cat: Console/PSP, desc: "GAMES PSP" }
+    - { id: 65, cat: Console/Wii, desc: "GAMES Wii" }
+    - { id: 15, cat: Console/XBox, desc: "GAMES Xbox" }
+    - { id: 21, cat: XXX/ImageSet, desc: "IMAGES Pics" }
+    - { id: 68, cat: XXX/ImageSet, desc: "IMAGES Wallpaper" }
+    - { id: 85, cat: Movies/Other, desc: "UpLINK HEVC" }
+    - { id: 86, cat: Movies/Other, desc: "UpLINK MPEG4" }
+    - { id: 70, cat: Movies/Other, desc: "Impatient CAM" }
+    - { id: 71, cat: Movies/Other, desc: "Impatient DVDSCR" }
+    - { id: 72, cat: Movies/Other, desc: "Impatient TC" }
+    - { id: 73, cat: Movies/Other, desc: "Impatient TS" }
+    - { id: 75, cat: Movies/Other, desc: "Impatient WEBRip" }
+    - { id: 26, cat: PC/Mobile-Android, desc: "PROGRAM Android" }
+    - { id: 27, cat: PC/Mobile-iOS, desc: "PROGRAM iOS" }
+    - { id: 78, cat: PC/ISO, desc: "PROGRAM Linux" }
+    - { id: 79, cat: PC/Mac, desc: "PROGRAM Mac" }
+    - { id: 9, cat: PC/ISO, desc: "PROGRAM PC ISO" }
+    - { id: 80, cat: PC/ISO, desc: "PROGRAM PC Win" }
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep]
+    movie-search: [q]
+    music-search: [q]
+    book-search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+  - name: password
+    type: password
+    label: Password
+  - name: freeleech
+    type: checkbox
+    label: Download only freeleech torrents?
+    default: false
+  - name: sort
+    type: select
+    label: Sort requested from site
+    default: 4
+    options:
+      0: torrent id
+      1: title
+      2: files
+      3: comments
+      4: date created
+      5: size
+      6: times completed
+      7: seeders
+      8: leechers
+  - name: type
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
+
+login:
+  path: login
+  method: form
+  form: form[action="/takeloginx.php"]
+  inputs:
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+  error:
+    - selector: i[class="ico-not_warn"] # if this icon is on the page, something went wrong
+  test:
+    path: /brskanje # browse page
+    selector: a[href*="/odjava?hash_insert="] # logout link, only present if you are logged in
+
+search:
+  paths:
+    # https://www.mojblink.si/brskanje/?cat=0&search=TEST&searchin=title&incldead=0&type=DESC&sort=4&only_free=1
+    # https://www.mojblink.si/brskanje/?c25=1&c46=1&search=TEST&searchin=descr&incldead=0&type=DESC&sort=4&only_free=0
+    # https://www.mojblink.si/xxx?c122=1&c88=1&c10=1&c89=1&c90=1&search=TEST&searchin=title&incldead=0
+    # TODO: if tracker finds only 1 matching results, it makes a redirect to the details page of torrent, then parsing fails and returns 0 results
```

Comment:
It was because of this that it was determined as not being suitable, and may need to be a C# indexer - https://github.com/Jackett/Jackett/issues/4090#issuecomment-805157978

If this were to be implemented in spite of that problem, there are a few other issues that will need to be fixed with this indexer.

Reply:
If I can provide the HTML output for browse and the detail page, would this help someone to prepare the C# code?

Reply:
If @garfield69's account is still active, they can share the login credentials for it, the issue is just getting a C# dev with the time to do it.